docs/book/src/reference/ports.md

```diff
@@ -2,7 +2,7 @@
 
 Name      | Port Number | Description |
 ---       | ---         | ---
-`metrics` |             | Port that exposes the metrics. This can be customized by setting the `--metrics-bind-addr` flag when starting the manager. The default is to only listen on `localhost:8080`
+`diagnostics-address` |             | Port that exposes the metrics, the pprof endpoint and an endpoint to change the log level. This can be customized by setting the `--diagnostics-address` flag when starting the manager. The default port is  `8443`. When `insecure-diagnostics` flag is used it is defaulted to port `8080`
```

Comment:
```suggestion
`diagnostics-address` |             | Port that exposes the metrics, the pprof endpoint and an endpoint to change the log level. This can be customized by setting the `--diagnostics-address` flag when starting the manager. The default port is  `8443`.
```
Just noticed now. I think the last part is not true